src/masternodes/govvariables/attributes.cpp

```diff
@@ -182,5 +250,24 @@ Res ATTRIBUTES::Validate(const CCustomCSView & view) const
 
 Res ATTRIBUTES::Apply(CCustomCSView & mnview, const uint32_t height)
 {
+    for (const auto& item : attributes) {
+        auto res = ProcessVariable(item.first, item.second, GetTypeByKey, GetTypeByKey,
+                                   [&](const UniValue& values) -> Res {
+            if (values[0].get_int() == AttributeTypes::Token) {
+                if (values[2].get_int() == TokenKeys::DexFeePCT) {
+                    auto res = VerifyPct(values[3].getValStr());
+                    if (!res) {
+                        return std::move(res);
+                    }
+                    auto tokenId = uint32_t(values[1].get_int());
+                    return mnview.SetDexfeePct({tokenId}, *res.val);
```

Comment:
I do not think this is correct. ATTRBIUTES is the store of data which should not be then stored separately like in other Gov vars. We just need a better way to access the map.

Reply:
We can parse and keep in `UniValue` i still prefer to keep some of them separately as `LP_SPLITS::Apply` or `ICX_TAKERFEE_PER_BTC::Apply`

Reply:
Actually all variables apply records to the particular view.